src/Framework/Framework/Compilation/Parser/Binding/Parser/BindingParser.cs

```diff
@@ -47,6 +47,75 @@ public BindingParserNode ReadDirectiveValue()
             return first;
         }
 
+        public BindingParserNode ReadArrayInitializerValue()
+        {
+            var startIndex = CurrentIndex;
+            SkipWhiteSpace();
+
+            if (Peek() is BindingToken openBraceToken && openBraceToken.Type == BindingTokenType.OpenArrayBrace)
+            {
+                var initializerExpressions = new List<BindingParserNode>();
+                Read();
+                SkipWhiteSpace();
+                initializerExpressions.Add(ReadArrayInitializerValue());
+                SkipWhiteSpace();
+
+                while (Peek() is BindingToken comma && comma.Type == BindingTokenType.Comma)
+                {
+                    Read();
+                    SkipWhiteSpace();
+                    initializerExpressions.Add(ReadArrayInitializerValue());
```

Comment:
I think you should also skip whitespace after the expression

Reply:
True, fixed